.github/workflows/manual-build.yaml

```diff
@@ -45,21 +48,29 @@ jobs:
         uses: docker/login-action@v1
         with:
           registry: quay.io
-          username: ${{ secrets.QUAY_LVMO_ROBOT_NAME_2 }}
-          password: ${{ secrets.QUAY_LVMO_ROBOT_TOKEN_2 }}
+          username: ${{ secrets.QUAY_LVMO_ROBOT_NAME }}
+          password: ${{ secrets.QUAY_LVMO_ROBOT_TOKEN }}
 
       - name: Get Current Date
         id: date
-        run: echo "::set-output name=date::$(date +'%Y%m%d')"
+        run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
 
       - name: Prepare image tag
         id: prep
         run: |
-          LVMO_OPERATOR_IMAGE=${{env.IMG}}:${{ github.event.inputs.branch }}-${{ steps.date.outputs.date }}
-          echo ::set-output name=operator-img::${LVMO_OPERATOR_IMAGE}
+          LVMO_OPERATOR_IMAGE=${{env.OPERATOR_IMG}}:${{ github.event.inputs.branch }}-${{ steps.date.outputs.date }}
+          LVMO_OPERATOR_BUNDLE=${{env.OPERATOR_BUNDLE_IMG}}:${{ github.event.inputs.branch }}-${{ steps.date.outputs.date }}
+          echo "operator-img=${LVMO_OPERATOR_IMAGE}" >> $GITHUB_OUTPUT
+          echo "bundle-img=${LVMO_OPERATOR_BUNDLE}" >> $GITHUB_OUTPUT
 
       - name: Build operator docker image
         run: make docker-build-combined IMG=${{steps.prep.outputs.operator-img}}
 
       - name: Publish operator image
         run: make docker-push IMG=${{steps.prep.outputs.operator-img}}
+
+      - name: Build operator bundle image
+        run: make bundle bundle-build BUNDLE_IMG=${{steps.prep.outputs.bundle-img}}
+
+      - name: Publish operator bundle image
+        run: make bundle-push BUNDLE_IMG=${{steps.prep.outputs.bundle-img}}
```

Comment:
I think build and push for a single image can come together.

Reply:
Done.